content/blog/2020-02-10-introduction-to-machine-learning-with-the-tidyverse/index.Rmarkdown

```diff
@@ -0,0 +1,191 @@
+```
+
+## [Prework](https://conf20-intro-ml.netlify.com/prework/)
+
+We set up [RStudio Server Pro](https://rstudio.com/products/rstudio-server-pro/) workspaces for all workshop attendees which provided more horsepower for running some of the more computationally intensive models, and which came pre-loaded with all the workshop exercises as R Markdown files and the packages needed to do them pre-installed. For those who wished to follow along locally, here is the list of additional packages needed:
+
+```{r eval = FALSE}
+install.packages(c("kknn", "rpart", "rpart.plot", "rattle", 
+                   "AmesHousing", "ranger", "partykit", "vip"))
+
+# and
+
+remotes::install_github("tidymodels/modeldata")
+```
+
+## Roadmap
+
+During the workshop, we presented slides interspersed with timed coding exercises called "Your Turns". 
+
+| session | slides                                 | Exercises + Solutions                   | 
+|---------|----------------------------------------|-----------------------------------------|
+| 00      | [Welcome & Introductions][00-slides]   |                                         |
+| 01      | [Predicting][01-slides]                | [Exercises][01-ex], [Solutions][01-sol] |
+| 02      | [Classifying][02-slides]               | [Exercises][02-ex], [Solutions][02-sol] |
+| 03      | [Sampling & Resampling][03-slides]     |                                         |
+| 04      | [Ensembling][04-slides]                | [Exercises][04-ex], [Solutions][04-sol] |
+| 05      | [Workflows][05-slides]                 | [Exercises][05-ex], [Solutions][05-sol] |
+| 06      | [Recipes][06-slides]                   | [Exercises][06-ex], [Solutions][06-sol] |
+| 07      | [Cross-validation][07-slides]          | [Exercises][07-ex], [Solutions][07-sol] |
+| 08      | [Tuning][08-slides]                    | [Exercises][08-ex], [Solutions][08-sol] |
+
+
+## Instructor Notes
+
+We did a trial run of this workshop in December 2019 in Boston with about 20 participants, which proved one of [Greg Wilson's cardinal rules](https://teachtogether.tech/#the-rules):
+
+> "Remember that no lesson survives first contact with learners…"
+
+Actually, the workshop went pretty smooth for a first run, and we received positive feedback from our attendees. But, like any good educators, Garrett and I decided that a content renovation would make the workshop **even better**. This decision was driven by a few observations:
+
+1. We realized that the process of using a fitted model object for generating predictions was pretty new to many attendees. We needed to spend more time on this, so we beefed up our [early section on predicting][01-slides] considerably. This meant that [parsnip](https://tidymodels.github.io/parsnip/) was our first tidymodels package to introduce, which felt right! This should be your first tidymodels package to learn/teach if you are new here (previously, we had started with [rsample](https://tidymodels.github.io/rsample/)).
```

Comment:
"...was the first we introduced" rather than "was our first package to introduce" (and similarly in the next phrase).